welcome.md

```diff
@@ -0,0 +1,111 @@
+# Welcome to Launchpad Lab
+
+We're excited to have you here. This packet contains information that past hires have found helpful.
+
+We recommend looking through it now and referring back to it later. Think of this packet as your Hitchhiker's Guide during your onboarding.
+
+Feel free to file a pull request to improve any aspect of it, no matter how tiny.
+
+# The Plan
+
+  Your first couple months will most likely be spent learning. Depending on what role you're assuming (Junior, Mid, Senior), you may or may not be expected to engage client work rather quickly. 
+  
+  Either way your onboarding helper is there to answer questions, pair with, and guide you through the acclimation process.
+
+## How am I doing?
+
+  You will meet with your onboarding helper once a week to talk through your progress and goals for the week, usually on Monday.
+
+  You are encouraged to create a list of goals you have for your onboarding, as well as maintain a weekly list of smaller, specific goals.
+
+  Generally, the goal is to become acclimated to your client project. However we welcome and encourage contribution to any of our internal projects and initiatives.
+
+## Pairing during the week
+
+  You will work with your helper on a client project throughout the week, sitting in on meetings and internal discussions. Your helper with pair with you during the week as you work on features.
+
+  Pairing is the fastest and most effective way to learn, so it should be a priority for you and your helper. However, do not expect to pair 100% of the time.
+
+  We also encourage you to pair with other members of the team (designer/developer). Everyone on the team loves teaching and learning, and they'd love to teach and learn from you. A good way to do it is to reach out on Slack that you're looking for someone to pair with, and if people have free time, they'll let you know.
+
+## Self-guided learning
+
+  Whenever appropriate or applicable to your work, you should take time away from client work to do some self-guided learning. This can include reading material from books, attending workshops, or learning about and trying out a technique or library.
+
+  For reading material, we recommend starting with 'Getting Naked' by Patrick Lencioni - which can be found on the library shelf.
+
+# Tools
+
+## Laptop setup
+
+  Your onboarding helper will assist getting your laptop set up. Generally, your work environment should support Ruby and Javascript development; if you are design oriented, this is not a requirement (but highly recommended!).
+  
+  We will allow you to use the tools you're most comfortable with, but we recommend this short list:
+
+  - Editors: VS Code, Atom, Sublime.
+  - Terminals: iTerm 2, Hyper, Terminal 
+  - Harvest (native app)
+  - Slack (native app)
+  - Postman (native app)
+  - Chrome
```

Comment:
@mrjonesbot Here is the [list](https://github.com/LaunchPadLab/process/blob/master/OnBoarding/machine-setup.md) we've used in the past. Feel free to pull these into this section. Or instead link out to that list and update the formatting.